calculator.py

```diff
@@ -0,0 +1,50 @@
+import calendar
+import sys
+import datastructures
+
+
+class Calculator:
+    def __init__(self):
+        self.weather_data: list[datastructures.WeatherData] = []
+
+    def find_highest_in_year(self):
```

Comment:
a function should do one thing and it should do it very well.

Reply:
this function is doing two things: calculating and prining. can we separate that